pycode/memilio-simulation/memilio/simulation_test/data/ode-secihurd-ageres-compare.csv

```diff
@@ -0,0 +1,37 @@
+ # t S E C C_confirmed I I_confirmed H U R D
+0.00000000000000  9760.00000000000000 99.99999999999999 49.99999999999999 0.00000000000000 49.99999999999999 0.00000000000000 20.00000000000000 10.00000000000000 10.00000000000000 0.00000000000000
```

Comment:
is this equal to the testdata from the cpp part?

Reply:
No, they are not due to using different integrators